### DISCUSSION TITLE
No puedo usar la version 7.2.5 de php. Error de apache

### DISCUSSION THREAD

**Discussion by MilerFD** [leokhoa/laragon#665] [Q&A]
Tengo problemas al usar la version de php 7.2.5
![capture_240121_210314](https://github.com/leokhoa/laragon/assets/107053345/b532db7a-5cec-4ba0-82e4-1b84b7e9fd0a)

Tengo entendito que se tiene que descargar la version compatible de apache "2.4.33" para php pero no logro encontrarlo,
solo encontro versiones
2.4.38,
2.4.41 y otros

Alguien que me pueda ayudar, Estare muy agradecido

**Comment by Pen-y-Fan** [comment]
See my blog [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/) for more information on the 

Apache and PHP need to have the same architecture 64-bit (x64) architecture and Visual Studio C++ run time (VC15).

I'm guessing you installed the wrong version of Apache, the correct version can be installed from [Apache lounge additional](https://www.apachelounge.com/download/additional/)
